internal/storage/migration/postgres/001.init_schema.sql

```diff
@@ -0,0 +1,11 @@
+-- +migration Up
+
+CREATE TABLE shot_url (
+    id         SERIAL  NOT NULL,
+    hash_value BIGINT NOT NULL,
```

Comment:
```suggestion
    hash_value smallserial NOT NULL,
```

Reply:
smallserial is too small to store uint32, perhaps you mean bigserial, bigserial does not use negative numbers, it will be more beneficial to use than bigint

Reply:
> 
smallserial is too small to store uint32, perhaps you mean bigserial, bigserial does not use negative numbers, it will be more beneficial to use than bigint